saleor/graphql/views.py

```diff
@@ -26,6 +29,27 @@
 handled_errors_logger = logging.getLogger("saleor.graphql.errors.handled")
 
 
+def tracing_wrapper(execute, sql, params, many, context):
+    if not settings.ENABLE_OPENTRACING:
+        return execute(sql, params, many, context)
+
+    start = time.time()
+    with opentracing.tracer.start_span(operation_name="query") as span:
```

Comment:
Maybe we could move `tracer` to `settings.py` to make it easier to customize?

For example the APM opentracing bridge requires this https://www.elastic.co/guide/en/apm/agent/python/4.x/opentracing-bridge.html

```python
from elasticapm.contrib.opentracing import Tracer

tracer = Tracer()
```

Reply:
```suggestion
    with opentracing.global_tracer().start_span(operation_name="query") as span:
```